ui/components/filters/autocompleter.js

```diff
@@ -45,7 +47,13 @@ export default class Autocompleter extends React.Component {
 
   render() {
     const { insertParam, lookup } = this.props;
-    const { pathname, query } = this.context.router.location;
+    const { query } = this.context.router.location;
+    let { pathname } = this.context.router.location;
+    // if we are coming from a page other than /requirements or
+    // /policies (ex. the homepage), default to going to /requirements
+    if (redirectWhiteList.includes(pathname) === false) {
+      pathname = '/requirements';
+    }
```

Comment:
I think there's confusion here over `redirectWhiteList`. When a user doesn't have JS, they're sent to a search results page whose links are populated by a set of `redirect` fields. As a security measure, we prevent search results from redirecting to _any_ page by using a whitelist.

So, the `pathname` variable should contain the _destination_ of the redirect. Thus far, the destination has been the same page, but that's not the case anymore. We probably want to check `this.props.pathname` instead of the current page (`this.context.router.location`).